src/components/structures/AutoHideScrollbar.js

```diff
@@ -23,6 +23,20 @@ export default class AutoHideScrollbar extends React.Component {
         this._collectContainerRef = this._collectContainerRef.bind(this);
     }
 
+    componentDidMount() {
+        if (this.containerRef && this.props.onScroll) {
```

Comment:
Should these be `this.containerRef.current`?

Reply:
It's a weird one, it's still using the old React way of collecting refs, but is also using the new naming... I'll probably update the code to align that with the way refs normally works